theano/sandbox/rng_mrg.py

```diff
@@ -1117,6 +1118,81 @@ def normal(self, size, avg=0.0, std=1.0, ndim=None,
         assert final_samples.dtype == dtype
         return final_samples
 
+    def truncated_normal(self, size, avg=0.0, std=1.0, ndim=None,
+                         dtype=None, nstreams=None):
```

Comment:
This would need to add support for the new `target` parameter: https://github.com/Theano/Theano/pull/6355

Reply:
@nouiz This is the review of abergeron I referred to.